### DISCUSSION TITLE
Montly Meetings

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#919] [Q&A] [answered]
Greetings

We have a monthly speaker meeting and am curious how we can list it just for the fourth Saturday of the month?

Thanks!

**Comment by kiyote33** [comment, marked as answer]
The 12 Step Meeting List is designed to list weekly meetings. If you have a regular meeting on the other Saturdays, you might mention your Speaker Meeting in the Notes field, much like many groups who note their birthday meetings on a particular week of the month.

**Reply by eflashlogistics** [reply]
This is just a speaker meeting that happens monthly. I do have a developer on my team and she just mentioned she will look into it and offer a patch to the developers if they want. She is also a new mom so unsure how soon that will happen ha.

**Reply by joshreisner** [reply]
Are you connected to Meeting Guide? App support recommends listing these meetings separately, or a separate page, or in a calendar, like a recurring event.

Solving this problem would be an interface- and system-design problem as much as a software problem, and not something that can be fixed quickly. Please let your software developer know that we'd be pleased to have her join our biweekly meetings, more info available at https://code4recovery.org/

**Reply by eflashlogistics** [reply]
Thanks. No connection to meeting guide, its al-anon. I just put a notice above the meeting listing to go to the events page for speaker meetings 

Will do. I get it that its just not designed that way.
